runconfig/config_windows.go

```diff
@@ -0,0 +1,13 @@
+package runconfig
+
+// ContainerConfigWrapper is a Config wrapper that hold the container Config (portable)
+// and the corresponding HostConfig (non-portable).
+type ContainerConfigWrapper struct {
+	*Config
+	HostConfig *HostConfig `json:"HostConfig,omitempty"`
```

Comment:
The use of innerhostconfig on Linux is really a gross hack for backwards compatibility. It really is a hostconfig in the Windows case, there is no back compat we need to keep there. I figured calling an apple an apple rather than a wrapper hack for a banana masquerading as an apple was cleaner :)